docs/chaos/latency.md

```diff
@@ -0,0 +1,142 @@
+# Latency monkey strategy
+
+## About
+
+- **Options**: [`LatencyStrategyOptions`](xref:Polly.Simmy.Latency.LatencyStrategyOptions)
+- **Extensions**: `AddChaosLatency`
+- **Strategy Type**: Proactive
+
+---
+
+The latency chaos strategy is designed to introduce controlled delays into system operations, simulating network latency or slow processing times. This strategy helps in assessing and improving the resilience of applications against increased response times.
+
+## Usage
+
+<!-- snippet: Latency -->
+```cs
+// Latency using the default options.
+// See https://www.pollydocs.org/chaos/latency#defaults for defaults.
+var optionsDefault = new LatencyStrategyOptions();
+
+// The 60% of calls will be randomly affected.
+var basicOptions = new LatencyStrategyOptions
+{
+    Latency = TimeSpan.FromSeconds(30),
+    Enabled = true,
+    InjectionRate = 0.6
+};
+
+// To use a custom function to generate the latency to inject.
+var optionsWithLatencyGenerator = new LatencyStrategyOptions
+{
+    LatencyGenerator = static args =>
+    {
+        TimeSpan latency = args.Context.OperationKey switch
+        {
+            "DataLayer" => TimeSpan.FromMilliseconds(500),
+            "ApplicationLayer" => TimeSpan.FromSeconds(2),
+            _ => TimeSpan.Zero // when the latency generator returns Zero the strategy won't inject any delay and it will just invoke the user's callback
```

Comment:
```suggestion
            _ => TimeSpan.Zero // When the latency generator returns Zero the strategy won't inject any delay and it will just invoke the user's callback
```

Reply:
Done